.travis.yml

```
@@ -3,6 +3,10 @@ sudo: true
 services:
   - docker
 
+# make comparing to origin/master work and fetch full history for correct current and previous version detection
+git:
+  depth: false
```

```suggestion
  depth: 500
```
I will be a long while until this becomes a problem in this repo anyway, but this aligns with Akka HTTP 
https://github.com/akka/akka-http/blob/41788d857c11a711eb07dff8b0ec1654b20e367e/.travis.yml#L8-L9